include/lbann/weights/weights.hpp

```diff
@@ -33,6 +33,7 @@
 #include "lbann/optimizers/optimizer.hpp"
 #include "lbann/io/persist.hpp"
 #include "lbann/utils/description.hpp"
+#include "lbann/utils/memory.hpp"
```

Comment:
Is this needed here?

Reply:
Nope.

Reply:
`std::unique_ptr` is used below -- you should `#include` something.

Reply:
We include `<memory>`, so I think we're good. As far as I can tell, `lbann/utils/memory.hpp` only include `make_unique`, which we don't use in this file or the associated source file.

In any case, things compile for me.

Reply:
Oh, yeah, the main "conversation" snippet didn't show that line and I missed it when I skimmed the file. 👍